skops/card/_model_card.py

```diff
@@ -11,6 +11,7 @@
 from typing import Any, Iterator, Literal, Protocol, Sequence, Union
 
 import joblib
+from fairlearn.metrics import MetricFrame
```

Comment:
We should not import fairlearn at the top level, since it would mean that it's a required dependency. It should be imported inside the corresponding method, similar to how you import pandas there.

However, you should not import them directly, but use the helper function we introduced, like here: https://github.com/skops-dev/skops/blob/b324b8d3689037eda6f1a6b1b9650e7eda7dab64/skops/card/_model_card.py#L1130